src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
     private String password;
```

Comment:
@password

Reply:
@password above class

Reply:
It doesn't look very correct that the annotation is over the class in this case.
It's better to have password above the field. But okay.